Makefile

```diff
@@ -1,4 +1,9 @@
 
 .PHONY: gen
 gen:
-	speakeasy generate sdk -s openapi.yaml -o . -d
\ No newline at end of file
+	@echo "Generating SDK"
+	curl -sSL https://insulator.conductor.one/api/v1/openapi.yaml > ./openapi.yaml
+	speakeasy generate sdk -s openapi.yaml -o . -d
+	rm openapi.yaml
+	@echo "Fixing Permissions"
+	find * -type f -perm '+rwx' | xargs chmod -x
```

Comment:
It'd be nice to leave a comment here explaining why we do this, so we can remove it when speakeasy fixes their CLI.